pkgs/initiator/initiator.go

```diff
@@ -287,13 +287,16 @@ func (c *Initiator) ReshareMessageFlowHandling(id [24]byte, signedReshare *wire.
 		reqIDs = append(reqIDs, msgID)
 	}
 	c.Logger.Info("sending signed reshare message to all operators")
-	var errs map[uint64]error
-	exchangeMsgs, errs, err := c.SendReshareMsg(id, signedReshare, allOps)
+	var initErrs map[uint64]error
+	exchangeMsgs, initErrs, err := c.SendReshareMsg(id, signedReshare, allOps)
```

Comment:
Possibly dumb question, and not very significant issue, if I am correct: we are sending the signed reshare message here, shouldn't `initErrs` be called `reshareErrs` instead?

Reply:
I see you call this "init phase" below, so I guess there's that. 🤔